modulemd/v2/include/modulemd-2.0/modulemd-module.h

```diff
@@ -33,6 +33,18 @@ G_DECLARE_FINAL_TYPE (
   ModulemdModule, modulemd_module, MODULEMD, MODULE, GObject)
 
 
+/**
+ * modulemd_module_get_stream_names
+ * @self: This #ModulemdModule
+ *
+ * Returns: (transfer full): An ordered list of stream names in this module.
```

Comment:
Please don't use a GPtrArray here. When returning an array of strings, the libmodulemd public API prefers `GStrv` format (which is a NULL-terminated array of `gchar *` pointers).

You can use [`modulemd-ordered-str-keys-as-strv()`](https://fedora-modularity.github.io/libmodulemd/latest/modulemd-2.0-Modulemd-Utility-Functions.html#modulemd-ordered-str-keys-as-strv) to get that from a GHashTable.

Also, for the record, you wouldn't want to do `(transfer full)` if you were returning a `GPtrArray` here, because the `GPtrArray` type owns its own contents, so you'd be causing it to do a double-free. Also also, if you return a `GPtrArray`, you need to use the `(element-type ...)` directive in the gtkdoc to indicate what type the elements of the array is carrying. Without this, the python binding can't figure it out.